### PR TITLE
Add a new doc for Ansible deployment

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@LinuxGit @liubo0127 @iamxy PTAL. Please be noted that this is not exactly a word-to-word translation.

File: op-guide/ansible-deployment.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+title: TiDB Ansible Deployment
+category: deployment
+---
+
+# TiDB Ansible Deployment
+
+## Overview
+Ansible is an IT automation tool. It can configure systems, deploy software, and orchestrate more advanced IT tasks such as continuous deployments or zero downtime rolling updates.
+
+[TiDB-Ansible](https://github.com/pingcap/tidb-ansible) is a TiDB cluster deployment tool developed by PingCAP, based on Anisible playbook. TiDB-Ansible enables you to quickly deploy a new TiDB cluster which includes PD, TiDB, TiKV, and the cluster monitoring modules.a
```
Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: based on Anisible playbook:  "Ansible typo"

Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: redundant "a" in modules.a.

File: op-guide/ansible-deployment.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+- Access to the managed nodes via SSH using password login or SSH authorized_key login. 
+	
+
+1.2  Several managed nodes with the following requirements:
+
+- 4 or more machines. At least 3 instances for TiKV. Don’t deploy TiKV together with TiDB or PD on the same machine. See [deploying recommendations](https://github.com/pingcap/docs/blob/master/op-guide/recommendation.md).
+   
+- Operating system:
+   		
+	- Ubuntu, macOS, CentOS 7 or later, or Docker
```
Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: Recommand CentOS 7.2 or later Linux, not including macOS.

File: op-guide/ansible-deployment.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+	   		
+	- Kernel version 3.10 or later
+			
+	- Ext4 file system. 
+
+- Network between machines. Turn off the firewalls and iptables when deploying and turn them on after the deployment.
+
+- The same time and time zone for all machines with the NTP service on to synchronize the correct time.
+
+- A deploy account to deploy TiBD-Ansible. It can be the root user or a user account with sudo privileges.
```
Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: It is ambiguous.

File: op-guide/ansible-deployment.md [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+	  ansible_user = tidb
+	  ```
+	5.2 Connect to the network and download the TiDB, TiKV, and PD binaries:
+		
+		ansible-playbook local_prepare.yml
+	                                           
+	5.3 Initialize the system environment of the target machines and modify the kernel parameters:
+	
+		ansible-playbook bootstrap.yml -k -K
+
```
Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: add -k parameter note.

File: op-guide/ansible-deployment.md [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+	
+	5.3 Initialize the system environment of the target machines and update the kernel parameters
+	 
+		ansible-playbook bootstrap.yml -k
+		
+	**Note:** If the service user does not exist, the initialization operation will automatically create the user.
+	 
+	5.4 Change the root user to the normal user.
+	
+	Because root user is recommended to run the TiDB service, you need to change the root user to a normal user. Edit the `inventory.ini` file to uncomment `# ansible_become = true` as follows:
```
Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: root user is not recommended

File: op-guide/ansible-deployment.md [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+pd_servers
+ 
+[monitoring_servers]
+172.16.10.1
+ 
+[grafana_servers]
+172.16.10.1
+```
+ 
+### 5. Deploy TiDB Ansible
```
Comment (LinuxGit) [inline op-guide/ansible-deployment.md]: Deploy TiDB

**Pull request merged**